docs/BR.md

```diff
@@ -1272,55 +1279,64 @@ No stipulation.
+- the value of the `nextUpdate` field MUST NOT be more than twelve months beyond the value of the `thisUpdate` field.
 
 ### 4.9.8 Maximum latency for CRLs (if applicable)
 
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
 
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
+The CA MAY support on-line revocation status checking using OCSP.
+
+If the CA supports OCSP, the following requirements SHALL apply:
 
+OCSP responses MUST conform to RFC6960 and/or RFC5019; and MUST either: 
 1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
+2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose revocation status is being checked.
 
-In the latter case, the OCSP signing Certificate MUST contain an extension of type `id-pkix-ocsp-nocheck`, as
-defined by RFC6960.
+In the latter case, the OCSP signing Certificate MUST contain an extension of type `id-pkix-ocsp-nocheck`, as defined by RFC6960.
 
 ### 4.9.10 On-line revocation checking requirements
 
+The CA MAY support on-line revocation status checking using OCSP. 
+
+If the CA supports OCSP, the following requirements SHALL apply:
+
 OCSP responders operated by the CA SHALL support the HTTP GET method, as described in RFC 6960 and/or RFC 5019.
 
 The validity interval of an OCSP response is the difference in time between the `thisUpdate` and `nextUpdate` field, inclusive. For purposes of computing differences, a difference of 3,600 seconds shall be equal to one hour, and a difference of 86,400 seconds shall be equal to one day, ignoring leap-seconds.
 
 For the status of Subscriber Certificates:
 
-1. OCSP responses MUST have a validity interval greater than or equal to eight hours;
-2. OCSP responses MUST have a validity interval less than or equal to ten days;
-3. For OCSP responses with validity intervals less than sixteen hours, then the CA SHALL update the information provided via an Online Certificate Status Protocol prior to one-half of the validity period before the nextUpdate.
-4. For OCSP responses with validity intervals greater than or equal to sixteen hours, then the CA SHALL update the information provided via an Online Certificate Status Protocol at least eight hours prior to the nextUpdate, and no later than four days after the thisUpdate.
-
-For the status of Subordinate CA Certificates:
+- Prior to 2020-09-30:
+     1. The CA SHALL update information provided via an Online Certificate Status Protocol at least every four days. 
+     2. OCSP responses from this service MUST have a maximum expiration time of ten days.
 
-* The CA SHALL update information provided via an Online Certificate Status Protocol
+- Effective 2020-09-30:
```

Comment:
On #402, @BenWilson-Mozilla commented:

```Should this be shortened, too?```

Open for community discussion. This proposal does not change the existing OCSP requirements, it merely makes OCSP optional.

Reply:
Closing stale comments.